docs/source/examples/plot_toll_analysis.py

```diff
@@ -0,0 +1,170 @@
+"""
+Toll analysis with Select Link
+==============================
+
+On this example we show how to perform traffic assignment considering fixed
+link costs (e.g. tolls) besides travel time. We also do assignment with
+three different vehicle classes
+"""
+
+import logging
+import sys
+import urllib
+import zipfile
+from os.path import join
+
+## Imports
+from tempfile import gettempdir
+
+# %%
+from aequilibrae import logger, Project
+from aequilibrae.paths import TrafficAssignment, TrafficClass
+
+# We the project open, we can tell the logger to direct all messages to the terminal as well
+stdout_handler = logging.StreamHandler(sys.stdout)
+formatter = logging.Formatter("%(asctime)s;%(name)s;%(levelname)s ; %(message)s")
+stdout_handler.setFormatter(formatter)
+logger.addHandler(stdout_handler)
+
+# %%
+
+disk_pth = join(gettempdir(), "sioux_falls_multi-Class")
+urllib.request.urlretrieve("https://www.aequilibrae.com/data/sioux_falls_multi-Class.zip", disk_pth + ".zip")
+
+with zipfile.ZipFile(disk_pth + ".zip") as zf:
+    zf.extractall(gettempdir())
```

Comment:
This extracts files to the '/tmp' folder on my machine, it doesn't put them in '/tmp/sioux_falls_multi-Class/', so it fails on the following line `project.open(disk_pth)` (the folder doesn't exist).
Should it be `zf.extractall(disk_pth)` instead ?